packages/yourloops/components/patient-list/current-patient-list/current-private-team-or-caregiver-patient-list/current-private-team-or-caregiver-patient-list.tsx

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (c) 2023, Diabeloop
+ *
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import React, { type FunctionComponent, useState } from 'react'
+import { type Patient } from '../../../../lib/patient/models/patient.model'
+import { useCurrentPrivateTeamOrCaregiverPatientListHook } from './current-private-team-or-caregiver-patient-list.hook'
+import Box from '@mui/material/Box'
+import { DataGrid, type GridPaginationModel, type GridSortModel, useGridApiRef } from '@mui/x-data-grid'
+import { EmptyPatientList } from '../../empty-patient-list/empty-patient-list'
+import { PatientListCustomFooter } from '../../patient-list-custom-footer'
+import RemoveDirectShareDialog from '../../../dialogs/remove-direct-share-dialog'
+import { usePatientListContext } from '../../../../lib/providers/patient-list.provider'
+import { usePatientContext } from '../../../../lib/patient/patient.provider'
+import { useWindowDimensions } from '../../../../lib/custom-hooks/use-window-dimensions.hook'
+import { PatientListColumns } from '../../models/enums/patient-list.enum'
+import RemovePatientDialog from '../../../patient/remove-patient-dialog/remove-patient-dialog'
+
+interface CurrentPrivateTeamOrCaregiverPatientListProps {
+  patients: Patient[]
+}
+
+export const CurrentPrivateTeamOrCaregiverPatientList: FunctionComponent<CurrentPrivateTeamOrCaregiverPatientListProps> = (props: CurrentPrivateTeamOrCaregiverPatientListProps) => {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Don't you think that this naming is a bit of a overkill? Caregivers and private teams do not have access to the pending tab. Maybe it could be `PrivateTeamOrCaregiverPatientList` (or even `PrivateTeamPatientList` but I would understand that you wouldn't like this one :wink: )